packages/clarity-js/src/data/metadata.ts

```diff
@@ -19,32 +20,35 @@ export function start(): void {
   // Populate ids for this page
   let s = session();
   let u = user();
-  data = {
-    projectId: config.projectId || hash(location.host),
-    userId: u.id,
-    sessionId: s.session,
-    pageNum: s.count
-  }
+  let projectId = config.projectId || hash(location.host);
```

Comment:
Hashing the host name seems weird. In what cases would we do this?

Reply:
This is not new code and this is mostly for dev tools extension so it can work against any website as part of the testing.